ops/framework.py

```diff
@@ -878,7 +882,24 @@ def _reemit(self, single_event_path: str = None):
             observer = self._observer.get(observer_path)
             if observer:
                 if single_event_path is None:
-                    logger.debug("Re-emitting %s.", event)
+                    logger.debug("Re-emitting deferred %s.", event)
+                elif isinstance(event, FrameworkEvent):
+                    pass  # We get two of these each hook execution: too much noise.
```

Comment:
Indeed, the intention was to filter out all Framework.on events as those aren't really 'public' and should therefore not be of interest to the user, and because they're always there and therefore only cause noise. I'll rework the logic to make it more obvious what it is we're suppressing.